zebra/zebra_fpm.c

```diff
@@ -528,6 +533,14 @@ static int zfpm_conn_up_thread_cb(struct thread *thread)
 		goto done;
 	}
 
+	if (!zfpm_g->fpm_mac_dump_done) {
+		/* Enqueue FPM updates for all the RMAC entries */
+		hash_iterate(zrouter.l3vni_table, zfpm_iterate_rmac_table,
+			     NULL);
+		/* mark dump done so that its not repeated after yield */
+		zfpm_g->fpm_mac_dump_done = true;
+	}
+
 	/* Enqueue FPM updates for all the RMAC entries */
 	hash_iterate(zrouter.l3vni_table, zfpm_iterate_rmac_table, NULL);
```

Comment:
this is the existing iteration, yes? so ... now we'll iterate even more times?

Reply:
Thanks for review, my bad I missed to remove this code.